src/blockchain_poc_core_v1.proto

```diff
@@ -49,10 +49,10 @@ message blockchain_poc_witness_v1 {
   // NOTE: reward_unit and hex_scale are ignored when signing the receipt txn
   // integer representation of a 4-point precision decimal multiplier
   // based on the number of witnesses to a poc event
-  uint32 reward_unit = 11;
+  optional uint32 reward_unit = 11;
```

Comment:
you can't do these.. not supported by all but the latest protoc compilers. The "default value" is how optional is done for proto3 primitives

Reply:
Uh, alright then